README.md

```diff
@@ -30,4 +30,4 @@ The server will rebuild the site every time a file changes, with the exception o
 
 ## License
 
-© 2020 Foundation for Public Code
+© 2020 Foundation for Public Code. All content licensed under [Creative Commons Zero v1.0 Universal, unless otherwise noted](LICENSE.md).
```

Comment:
```suggestion
© 2020 Foundation for Public Code. All content licensed under [Creative Commons Zero v1.0 Universal, unless stated otherwise](LICENSE.md).
```

Reply:
I don't disagree about your wording @clausmullie, but I intentionally used the same phrasing as in the footer of the Jekyll theme. Should we perhaps update the theme too?

Reply:
Let's move ahead with the footer phrasing.